pkg/kapp/resources/identified_resources_list.go

```diff
@@ -47,6 +47,15 @@ func (r IdentifiedResources) List(labelSelector labels.Selector, resRefs []Resou
 		return nil, err
 	}
 
+	// Filter resources by label in case of aggregated api https://github.com/vmware-tanzu/carvel-kapp/issues/160
```

Comment:
"in case of aggregated api" feels like sentence didnt end. how about:

```
// Check returned resources against label selector 
// in case of Kubernetes APIs returned resource that do not match.
// This can happen if custom aggregated APIs did not implement label selector filtering.
// (https://github.com/vmware-tanzu/carvel-kapp/issues/160)
```